tests/testthat/indention_operators/if-else-no-braces-not-strict-out.R

```diff
@@ -1,17 +1,24 @@
-if (TRUE) c(
-  2
-) else c(
-  1
-)
+if (TRUE)
```

Comment:
The line break in `c()` is persistent. We'll add braces soon to make this less weird.